packages/gatsby-plugin-manifest/src/common.js

```diff
@@ -1,6 +1,14 @@
 import fs from "fs"
 import sysPath from "path"
 
+exports.favicons = [
+  {
+    src: `favicon-32x32.png`,
+    sizes: `32x32`,
+    type: `image/png`,
+  },
+]
+
```

Comment:
Although there's currently just one favicon being generated, I thought it would make sense to define this as an array to mirror `icons`. Means adding more sizes is easy as a bonus, possibly a case of YAGNI though 🤷‍♂️